lightseq/csrc/pybind/pybind_model.cpp

```diff
@@ -7,19 +7,20 @@
 #include "util.h"
 
 #include "bert.h"
-#include "bert_crf.h"
+// #include "bert_crf.h"
```

Comment:
why delete this model?

Reply:
Need to be modified to adapt to the interface code.